chatter/src/components/Messages/components/Message.js

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
 import cx from 'classnames';
+import { MY_USER_ID } from '../../UserList/constants/users';
 
-const ME = 'me';
 
-export default function Message({ nextMessage, message, botTyping }) {
+const Message = React.memo(({ message, isLast }) => {
```

Comment:
Refactored this `Message` component to avoid unnecessary re-rendering when `botTyping` changes.